src/rocm_docs/projects.py

```diff
@@ -295,9 +295,9 @@ def _update_theme_configs(
     app: Sphinx, current_project: Optional[_Project], current_branch: str
 ) -> None:
     """Update configurations for use in theme.py"""
-    latest_version = "5.6.1"
+    latest_version = "5.7"
```

Comment:
Is latest version supposed to be 5.7.0?

Reply:
Good point. Changed to 5.7.0 and 6.0.0 to include patch numbers.

The version of rocm-docs-core that this will create (currently expecting 0.24.0) will be cherry picked into the release/rocm-rel-5.7 branch